benchmark/benchmark.go

```diff
@@ -92,54 +92,131 @@ func (s *testServer) StreamingCall(stream testpb.TestService_StreamingCallServer
 	}
 }
 
-// StartServer starts a gRPC server serving a benchmark service on the given
-// address, which may be something like "localhost:0". It returns its listen
-// address and a function to stop the server.
-func StartServer(addr string) (string, func()) {
-	lis, err := net.Listen("tcp", addr)
+// byteBufServer is a gRPC server that sends and receives byte buffer.
+// The purpose is to remove the protobuf overhead from benchmark tests.
```

Comment:
The purpose is to benchmark the gRPC performance without protobuf serialization/deserialization overhead.